### ISSUE TITLE
having an error on upgrading from 18.04 to 22.04 on setup functions.sh

### ISSUE THREAD

**Issue by nyj1911** [mail-in-a-box/mailinabox#2172]
![image](https://user-images.githubusercontent.com/57099395/195348059-e87ab230-fc47-4ee9-9268-34f1ca862b67.png)

does anyone facing this issue? thank for the help

**Comment by Cmoi20** [comment]
hello,
I've update old mailinabox updated to v57 on ubuntu 18.04 updated to ubuntu 20.04 and to ubuntu 22.04 with do-release-update (need to modify /etc/update-manager/release-upgrades to activate lts upgrade)
rerun installation script. I've got issue explain a this url : https://forum.kopano.io/topic/3885/z-push-with-zimbra-backend-php-8-0-not-working/9
solved by message from umgfoin at 2021 dec, 17 in last link.
I've got same issue as you. solved by remove "/usr/local/lib/mailinabox" folder
after a last reboot, all is running well
